src/cli.rs

```diff
@@ -399,7 +400,20 @@ fn list_peers(peer_manager: Arc<PeerManager>) {
 	println!("\t}},");
 }
 
-fn list_channels(channel_manager: &Arc<ChannelManager>) {
+/// Takes some untrusted bytes and returns a sanitized string that is safe to print
+fn sanitize_string(bytes: &[u8]) -> String {
+	let mut ret = String::with_capacity(bytes.len());
+	// We should really support some sane subset of UTF-8 here, but limiting to printable ASCII
+	// instead makes this trivial.
+	for b in bytes {
+		if *b >= 0x20 && *b <= 0x7e {
+			ret.push(*b as char);
+		}
+	}
+	ret
+}
```

Comment:
It looks like using it would be more code than not using it. It only operates on a single byte, at a time, there's no "escape a byte array", and we'd have to map the nulls back to empty.